CSS/Indentation Rules.tmPreferences

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<plist version="1.0">
+<dict>
+	<key>scope</key>
+	<string>source.css</string>
+	<key>settings</key>
+	<dict>
+		<key>decreaseIndentPattern</key>
+		<string>^\s*[)\]}]</string>
+		<key>increaseIndentPattern</key>
+		<string>^.*[(\[{](?:\s*/\*.*\*/)*\s*(?:/\*.*)?$</string>
```

Comment:
do you think it is worth making it an extended regex so we can have it on multiple lines for readability?